scripts/gen-tag-table.py

```diff
@@ -689,6 +698,9 @@ def get_name(self, lt):
 bcp_47.names['mhv'] = 'Arakanese'
 bcp_47.scopes['mhv'] = '(retired code)'
 
+ot.remove_language_ot('MONT')
+ot.add_language('mnw', 'MONT')
+
```

Comment:
This doesn't appear in the original set of Harfbuzz changes, what is this for?

Reply:
The current version of [Microsoft’s page of language tags](https://docs.microsoft.com/en-us/typography/opentype/spec/languagetags) has a note for Thailand Mon that chokes the parser. I had to add this for the script to finish successfully.